docs/tools/vdb_table/data/rockset.json

```diff
@@ -14,91 +14,91 @@
     "comment": ""
   },
   "license": {
-    "value": "",
+    "value": "Proprietary",
     "source_url": "",
     "comment": ""
   },
   "dev_languages": {
     "value": [
-      ""
+      "SQL"
     ],
     "source_url": "",
     "comment": ""
   },
-  "github_stars": 0,
-  "vector_launch_year": 0,
+  "github_stars": "Proprietary",
+  "vector_launch_year": 2023,
   "metadata_filter": {
-    "support": "",
+    "support": "full",
     "source_url": "",
-    "comment": ""
+    "comment": "Proprietary"
   },
   "hybrid_search": {
-    "support": "",
+    "support": "full",
```

Comment:
all claims of full support should come with direct links to code or documentation that reflect that feature, added to source_url (applies throughout the PR)

Reply:
added two links in there, one of which actually has queries that support hybrid search.